.github/workflows/brain-games-workflow.yml

```diff
@@ -0,0 +1,27 @@
+name: lint
```

Comment:
rename this file to `lint.yml` (don't forget to update the commit message and badge link in the README.md)